src/iptvsimple/PlaylistLoader.h

```diff
@@ -27,6 +27,7 @@ namespace iptvsimple
   static const std::string TVG_INFO_LOGO_MARKER    = "tvg-logo=";
   static const std::string TVG_INFO_SHIFT_MARKER   = "tvg-shift=";
   static const std::string TVG_INFO_CHNO_MARKER    = "tvg-chno=";
+  static const std::string TVG_INFO_REC            = "tvg-rec="; //some provider use 'tvg-rec' instead of 'catchup-days'
```

Comment:
`// some providers use 'tvg-rec' instead of 'catchup-days'`

Reply:
Copy paste is bad, roger ;)